numpy/_core/src/multiarray/nditer_api.c

```diff
@@ -1901,21 +1935,17 @@ npyiter_copy_from_buffers(NpyIter *iter)
 
     npyiter_opitflags *op_itflags = NIT_OPITFLAGS(iter);
     NpyIter_BufferData *bufferdata = NIT_BUFFERDATA(iter);
-    NpyIter_AxisData *axisdata = NIT_AXISDATA(iter),
-                    *reduce_outeraxisdata = NULL;
+    NpyIter_AxisData *axisdata = NIT_AXISDATA(iter);
+    NpyIter_AxisData *outer_axisdata = NULL;
 
     PyArray_Descr **dtypes = NIT_DTYPES(iter);
-    npy_intp transfersize = NBF_SIZE(bufferdata);
-    npy_intp *strides = NBF_STRIDES(bufferdata),
-             *ad_strides = NAD_STRIDES(axisdata);
-    npy_intp sizeof_axisdata = NIT_AXISDATA_SIZEOF(itflags, ndim, nop);
+    npy_intp *strides = NBF_STRIDES(bufferdata);
+    npy_intp transfersize = bufferdata->size;
```

Comment:
I wonder if it is better to keep using `NBF_SIZE(bufferdata)` here and elsewhere. Right now, it is becoming a mix, while before it seems to have been consistent.

Reply:
Yeah, maybe.  I really don't like the macro way where it's not necessary, but let me make it a bit more consistent at least.